src/timeline/components/stateChronicles/TimelineChronicleLegend.vue

```diff
@@ -156,6 +189,32 @@ export default {
         changeDisplayValue(value) {
             this.displayedValue = value;
         },
+        expandNumericHeight() {
+            this.numericHeight = this.numericHeight + 20;
+
+            const labelHeightInfo = {
+                name: this.chronicle.name,
+                index: this.index,
+                height: this.numericHeight + 4,
+            }
+
+            this.$emit('changeNumericHeight', labelHeightInfo);
+        },
+        decreaseNumericHeight() {
+            const newHeight = this.numericHeight - 20;
```

Comment:
Using a constant would remove the need to change this number in multiple places if we need to.